web/react/utils/utils.jsx

```diff
@@ -753,8 +753,23 @@ module.exports.isValidUsername = function(name) {
     return error;
 };
 
+function checkRTL(string) {
```

Comment:
We should think about using a library here.  I'm shocked there isn't built in support into JS for this.